install/install.php

```diff
@@ -263,7 +263,7 @@ function step3($host, $user, $password, $update) {
 
       //get database raw version
       $DB_ver = $link->query("SELECT version()");
-      $row = $DB_ver->fetch_array();
+      $row = $DB_ver->fetch(\PDO::FETCH_NUM);
```

Comment:
i saw some line like this, maybe we can add a `fetchArray` method ?

Reply:
In this case, we're dealing with a native PDOStatement we get from the query call.